game/src/js/scenes/GamepadScene.js

```diff
@@ -0,0 +1,91 @@
+import BaseScene from '../scenes/BaseScene';
+
+export default class GamepadScene extends BaseScene {
+  constructor() {
+    super({ key: 'GamepadScene', active: true });
+  }
+
+  preload() {
+    this.load.image('joystick', '/assets/sprites/joystick.png');
```

Comment:
Assets should be loaded when the game starts. All assets should be defined in `assets.json` so that they are both loaded when the game stars, and cached for offline mode.

I noticed this PR adds 3 assets, but only 2 are used. Maybe we can remove the unused one.

Reply:
I agree, however in this case the assets loaded from LoaderScene were not available for some reason. Hence I loaded them here... Have any idea why that would be? As for the additional asset, I accidentally left that in, I'll remove it.

Reply:
Weird. How were these "preloaded" assets being added to the scene? `this.add.image('joystick', x, y);` normally works on other scenes, but I haven't tried it on overlaid scenes.

Reply:
the same way, in fact I took 30 minutes messing around thinking it was something I was doing, I even tried using one of our other assets (that I know loads on other screens) and they came up as the green-no-image box as well unless I load them in this Scene. I'm baffled haha